From 5e3e056753d2bdd1e366a558e16854ce18a445ec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C3=ABl=20Arnauts?= <michael.arnauts@gmail.com>
Date: Tue, 18 Sep 2018 21:39:35 +0200
Subject: [PATCH] KEY_EPG is wrongly mapped to title instead of guide

It seems that `KEY_EPG` (what is labelled GUIDE on MCE remotes) is wrongly handled as `title`. This is probably a copy paste error from the past.
---
 system/Lircmap.xml | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/system/Lircmap.xml b/system/Lircmap.xml
index bd68c3937f1d..ec7c6e0e6a6c 100644
--- a/system/Lircmap.xml
+++ b/system/Lircmap.xml
@@ -449,7 +449,7 @@
 		<skipminus>KEY_PREVIOUSSONG</skipminus>
 		<skipminus>KEY_PREVIOUS</skipminus>
 		<title>KEY_TITLE</title>
-		<title>KEY_EPG</title>
+		<guide>KEY_EPG</guide>
 		<subtitle>KEY_SUBTITLE</subtitle>
 		<language>KEY_LANGUAGE</language>
 		<mute>KEY_MUTE</mute>
@@ -564,7 +564,8 @@
 		<pageminus>KEY_CHANNELDOWN</pageminus>
 		<skipplus>KEY_NEXT</skipplus>
 		<skipminus>KEY_PREVIOUS</skipminus>
-		<title>KEY_EPG</title>
+		<guide>KEY_EPG</guide>
+		<title>KEY_TITLE</title>
 		<subtitle>KEY_SUBTITLE</subtitle>
 		<language>KEY_LANGUAGE</language>
 		<info>KEY_INFO</info>
@@ -623,7 +624,8 @@
 		<pageminus>KEY_CHANNELDOWN</pageminus>
 		<skipplus>KEY_NEXT</skipplus>
 		<skipminus>KEY_PREVIOUS</skipminus>
-		<title>KEY_EPG</title>
+		<guide>KEY_EPG</guide>
+		<title>KEY_TITLE</title>
 		<subtitle>KEY_SUBTITLE</subtitle>
 		<language>KEY_LANGUAGE</language>
 		<info>KEY_INFO</info>
@@ -682,7 +684,8 @@
 		<pageminus>KEY_CHANNELDOWN</pageminus>
 		<skipplus>KEY_NEXT</skipplus>
 		<skipminus>KEY_PREVIOUS</skipminus>
-		<title>KEY_EPG</title>
+		<guide>KEY_EPG</guide>
+		<title>KEY_TITLE</title>
 		<subtitle>KEY_SUBTITLE</subtitle>
 		<language>KEY_LANGUAGE</language>
 		<info>KEY_INFO</info>
